source/css/sass/patterns/molecules/content-meta.scss

```diff
@@ -0,0 +1,30 @@
+@import "../../mixins/sizes";
+@import "../../mixins/typography";
+
+.content-meta {
+  @include remove-bullets-unordered-list();
+  @include blg-spacing("block-end", "small", "margin");
+  @include padding(0, "inline-start");
```

Comment:
Can removed as it's in a mixin (see also https://github.com/libero/pattern-library/pull/57#discussion_r268090823).